specs/takanon/common-knowledge.md

```diff
@@ -0,0 +1,7 @@
+"בניגוד לחוקים רגילים, התקנון לא מחייב אישור של נשיא המדינה או פרסום ברשומות כדי להיכנס לתוקף. זה מעניק לו מעמד ייחודי: הוא מסמך בעל תוקף משפטי פנימי שמסדיר את פעילות הכנסת, אך אינו כפוף ישירות לתהליכי החקיקה הסטנדרטיים."
```

Comment:
Why is this here now?

Reply:
was added by mistake directly in the repository.